docs/decisions/0019-selectors.rst

```diff
@@ -0,0 +1,4 @@
+Selectors for Dynamically Selecting Content
```

Comment:
I'm currently working on this section. Thanks!

Reply:
This is the first version of selectors based on the information that's available. I still haven't considered [these comments](https://github.com/openedx/openedx-learning/pull/251#discussion_r1845288014) so I'll be working on it.